### ISSUE TITLE
storage Manager for Runtime Node is blocking occasionally

### ISSUE THREAD

**Issue by jboulmal** [reTHINK-project/dev-service-framework#78]
Hi,
In Runtime Node we have parallel database connections from several and parallel node.js processes. Therefore, the database is in blocking states for concurrent read/write. Mainly here  [[StorageManager] - getVersion for key](https://github.com/reTHINK-project/dev-service-framework/blob/master/src/storage-manager/StorageManager.js#L71)
Thus, we need a way to handle exceptions during transactions. Especially, when the database is in blocking state. 
[i.e: here](https://github.com/dfahlander/Dexie.js/wiki/Dexie.on.blocked)
```
var db = new Dexie("MyDB");
db.on("blocked", function() {
    alert ("Database upgrading was blocked by another window/process. " );
   setTimeout("");
};
```

**Comment by pchainho** [comment]
it seems the usage of Dexie in nodejs is causing problems.

@dvilchez should we replace Dexie by something else like [PouchDB](https://pouchdb.com/learn.html)

We should give priority to this one

**Comment by jboulmal** [comment]
Solved here [70f3d2ceeb3cf3a712c5a678e503692168443394](https://github.com/reTHINK-project/dev-runtime-nodejs/commit/70f3d2ceeb3cf3a712c5a678e503692168443394)
